src/components/Menu/MenuItem/MenuItemButton/MenuItemButton.tsx

```diff
@@ -1,26 +1,29 @@
 import React, { FC } from 'react';
-import { MenuItemButtonProps } from '../MenuItem.types';
+import { MenuItemButtonProps, MenuItemIconAlign } from '../MenuItem.types';
 import { MenuSize, MenuVariant } from '../../Menu.types';
 import { mergeClasses } from '../../../../shared/utilities';
-import { Icon } from '../../../Icon';
+import { Icon, IconSize } from '../../../Icon';
 import { ButtonShape, ButtonSize, NeutralButton } from '../../../Button';
 
 import styles from '../menuItem.module.scss';
 
 export const MenuItemButton: FC<MenuItemButtonProps> = ({
-  iconProps,
-  text,
-  subText,
-  variant = MenuVariant.neutral,
-  size = MenuSize.medium,
+  active,
```

Comment:
also curious what is active used for in general? Just highlighting?

Reply:
styles